src/category_theory/groupoid/subgroupoid.lean

```diff
@@ -91,11 +130,17 @@ begin
   simp only [inv_eq_inv, is_iso.hom_inv_id],
 end
 
+lemma id_mem_of_src  {c d : C} {f : c ⟶ d} (h : f ∈ S.arrows c d) : (𝟙 c) ∈ S.arrows c c :=
+id_mem_of_nonempty_isotropy S c (mem_objs_of_src S h)
+
+lemma id_mem_of_tgt  {c d : C} {f : c ⟶ d} (h : f ∈ S.arrows c d) : (𝟙 d) ∈ S.arrows d d :=
+id_mem_of_nonempty_isotropy S d (mem_objs_of_tgt S h)
```

Comment:
```suggestion
lemma id_mem_of_src {c d : C} {f : c ⟶ d} (h : f ∈ S.arrows c d) : (𝟙 c) ∈ S.arrows c c :=
id_mem_of_nonempty_isotropy S c (mem_objs_of_src S h)

lemma id_mem_of_tgt {c d : C} {f : c ⟶ d} (h : f ∈ S.arrows c d) : (𝟙 d) ∈ S.arrows d d :=
id_mem_of_nonempty_isotropy S d (mem_objs_of_tgt S h)
```

Reply:
Sorry I thought I had corrected those: I think it might come from the fact that doing «Add suggestion to batch» and then «Resolve conversation» ends up removing the suggestion from the batch (in the GH interface).

Reply:
No big deal!
Also, after merging the suggestion, GitHub will automatically resolve the conversation, so there's no need to resolve the conversation yourself.